src/coreclr/jit/instr.cpp

```diff
@@ -877,7 +906,19 @@ void CodeGen::inst_RV_TT(instruction ins,
                     regSet.verifyRegUsed(regTmp);
                     return;
             }
-#else  // !TARGET_ARM
+#else // !TARGET_ARM
+#ifdef TARGET_LOONGARCH64
+            // For LoongArch64-ABI, the float arg might be passed by integer register,
+            // when there is no float register left but there is integer register(s) left.
+            if (emitter::isFloatReg(reg))
+            {
+                assert((ins == INS_fld_d) || (ins == INS_fld_s));
+            }
+            else if (emitter::isGeneralRegister(reg) && (ins != INS_lea))
+            {
+                ins = size == EA_4BYTE ? INS_ld_w : INS_ld_d;
```

Comment:
Maybe I have to keep this for LA right now  which is not dead code on LA.

Thanks

Reply:
The only use I see is in `genCodeForLclAddr` (that I assume was copied from ARM64), it can be easily replaced with direct emitter methods (like #65803 does for XARCH and ARM64). I think that would be better and will avoid introducing this strange code.

Reply:
Yes, you are right.
Now for the latest coreclr this is not used.

Thanks